htdocs/comm/propal/card.php

```diff
@@ -2849,8 +2849,8 @@
 		}
 
 		$tmparray = $object->getTotalWeightVolume();
-		$totalWeight = $tmparray['weight'];
-		$totalVolume = $tmparray['volume'];
+		$totalWeight = isset($tmparray['weight']) ?:0;
```

Comment:
Using isset with ?: is strange. It means it will return true if isset true instead of $weight

Reply:
Yes, error fixed